src/core/src/bootstrap/EnvLayer.py

```diff
@@ -501,6 +501,19 @@ def total_minutes_from_time_delta(time_delta):
         def total_seconds_from_time_delta(time_delta):
             return (time_delta.microseconds + (time_delta.seconds + time_delta.days * 24 * 3600) * 10 ** 6) / 10.0 ** 6
 
+        @staticmethod
+        def total_seconds_from_time_delta_round_to_one_decimal_digit(time_delta):
+            """ 
+            Converts the input time in datetime.timedelta format to seconds in float format
+            
+            Parameters:
+            time_delta (datetime.timedelta): time in datetime.timedelta format e.g. 0:00:00.219000
+            
+            Returns:
+            time in seconds round of to one decimal digit (float): e.g. 0.2 seconds
+            """
+            return round(EnvLayer.DateTime.total_seconds_from_time_delta(time_delta), 1)
```

Comment:
Non-blocker - @rane-rajasi this could have been a default parameter on the basic function, instead of an entire new function. Should be cleaned up later.